cmd/fyne/internal/commands/build.go

```diff
@@ -128,29 +130,42 @@ func checkGoVersion(runner runner, versionConstraint *version.ConstraintGroup) e
 	return checkVersion(string(goVersion), versionConstraint)
 }
 
-func (b *Builder) build() error {
-	var versionConstraint *version.ConstraintGroup
+type GoModEdit struct {
+	Module struct {
+		Path string
+	}
+	Require []struct {
+		Path    string
+		Version string
+	}
+}
 
-	goos := b.os
-	if goos == "" {
-		goos = targetOS()
+func getFyneGoModVersion(runner runner) (string, error) {
+	dependenciesOutput, err := runner.runOutput("mod", "edit", "-json")
```

Comment:
Another possible approach could be to use the https://pkg.go.dev/golang.org/x/mod@v0.5.1/modfile#Parse

Reply:
Yes, I started with that, but couldn't figure out an API that find the go.mod file properly. The call to `go mod edit -json` solved that problem.